accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/dumpservice/AcDumpElementYamlVisitor.java

```diff
@@ -33,21 +38,21 @@ public AcDumpElementYamlVisitor(final int mapOrder, final StringBuilder sb) {
     @Override
     public void visit(final AuthorizableConfigBean authorizableConfigBean) {
         sb.append(AcHelper.getBlankString(DUMP_INDENTATION_KEY))
-                .append("- " + authorizableConfigBean.getPrincipalID() + ":")
-                .append("\n");
+        .append("- " + authorizableConfigBean.getPrincipalID() + ":")
```

Comment:
Lots of formatting changes... isn't the old version better? (probably the auto-formatting has to be fixed)